Source/SwiftLintFramework/Extensions/SwiftExpressionCallKind.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SwiftExpressionCallKind.swift
+//  SwiftLint
+//
+//  Created by Marcelo Fabri on 11/12/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+
+public enum SwiftExpressionCallKind: String {
```

Comment:
We could rename and add the other members, but we either:
a) will have to keep the `other` case to keep the rules working
b) change [validateFile](https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Protocols/ASTRule.swift#L22-L32) implementation to look into the subitems no matter the `KindType`.